chuffed/vars/int-var.h

```diff
@@ -120,6 +120,14 @@ class IntVar : public Var {
 	void setPreferredVal(PreferredVal p) { preferred_val = p; }
 	DecInfo* branch();
 
+    // Solution-based phase saving
+    bool SBPSValueSelection;
```

Comment:
Please rename it and all its occurrences to "sbps_value_selection" to be consistent with existing naming convention.